### ISSUE TITLE
Where the pictures?

### ISSUE THREAD

**Issue by drjames5** [Chaxiraxi/DebugHUD#1]
Bad mod design to have a mod that changes something visually with no pictures!

**Comment by Chaxiraxi** [comment]
Yep, I agree a simple screenshot would be better. However the recent update of the game broke my mod and I currently do not have much time in my private life to fix it. I'll add a screenshot when the mod will work again.
________________________________
De : drjames5 ***@***.***>
Envoyé : vendredi 20 octobre 2023 18:36
À : Chaxiraxi/DebugHUD ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [Chaxiraxi/DebugHUD] Where the pictures? (Issue #1)


Bad mod design to have a mod that changes something visually with no pictures!
